src/components/StyleGuide/Typography.mdx

```diff
@@ -1252,6 +1316,298 @@ The following design tokens are available through the DS.
 | **Semibold** | `semibold` | `var(--nypl-fontWeights-semibold)` | `600` |
 | **Bold**     | `bold`     | `var(--nypl-fontWeights-bold)`     | `700` |
 
+### Thin
+
+Reserved for use only in the Heading component.
+
+{
+
+<Box
+  border="1px"
+  borderColor="ui.border.default"
+  borderRadius="5px"
+  mb="m"
+  p="s"
+>
+  <Heading level="h2" border="0">
+    Lorem Ipsum
+  </Heading>
+</Box>
+}
+
+### Light
+
+The default font weight for the Reservoir Design System typography.
+
+{
+
+<Box
+  border="1px"
+  borderColor="ui.border.default"
+  borderRadius="5px"
+  mb="m"
+  p="s"
+>
+  <Heading level="h2" border="0">
+    <>
+      Lorem{" "}
+      <Box as="span" fontWeight="light">
+        Ipsum
+      </Box>
+    </>
+  </Heading>
+  <p
+    style={{
+      fontSize: "var(--nypl-fontSizes-0)",
+      fontWeight: "var(--nypl-fontWeights-light)",
+      fontFamily: "var(--nypl-fonts-body)",
+    }}
```

Comment:
If "light" is the default, why add it explicitly? Because Storybook?

Reply:
Yes, Storybook 7 is introducing styles that are causing problems with our default text styles, so I added inline styles to get the text to look correct.